src/Traits/Http.php

```diff
@@ -253,15 +267,17 @@ protected function prepareMultipartParams(array $params, $inputFileField): array
 
         $inputFile = $params[$inputFileField];
 
-
+        //All files should be provided in an InputFile object
         if (is_resource($inputFile)) {
             throw CouldNotUploadInputFile::resourceShouldBeInputFileEntity($inputFileField);
         }
 
+        //If the user provides a path to a file, attempt to create an InputFile Object automatically for them.
         if (is_string($inputFile)) {
             $params[$inputFileField] = InputFile::create($inputFile);
         }
```

Comment:
Are we getting rid of these 2 checkers and instead just have one to check if it's an instance of InputFile?